zero_stock_blockage/__manifest__.py

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "zero_stock_blockage",
```

Comment:
Avoid technical name here
```suggestion
    "name": "Zero stock blockage",
```